store/postgres/src/chain_store.rs

```diff
@@ -1413,6 +1432,16 @@ impl ChainStoreTrait for ChainStore {
             .block_number(&conn, hash)?
             .map(|number| (self.chain.clone(), number)))
     }
+
+    fn transaction_receipts_in_block(
```

Comment:
Store operations are blocking. Since this is called in an async context, it should be made async. You can accomplish this by using `with_conn` instead of `get_conn`.